plugins/chat/spec/system/page_objects/chat/components/message.rb

```diff
@@ -7,7 +7,7 @@ class Message < PageObjects::Components::Base
         attr_reader :context
         attr_reader :component
 
-        SELECTOR = ".chat-message-container:not(.has-thread-indicator)"
+        SELECTOR = ".chat-message-container"
```

Comment:
Having this in the selector is a mistake here, we should still be able to check a message if it includes a thread indicator. Will look if it breaks anything.